scripts/beck/st/error_injection.py

```diff
@@ -167,27 +171,35 @@ def ReadTypeSize(byte):  # Ex: 0001(ver) 0009(type) 00003dd8(size)
 
 
 def EditMetadata(offset):
+    """
+    Edit metadata with fix pattern of 0x1111222244443333
+    """
     with open(filename, 'r+b') as wbfr:
-        logger.info("** Corrupting 8byte of Metadata at offset {} with b'1111222244443333' **".format(offset))
+        logger.info("** Corrupting 8byte of Metadata at offset {}"
+                    " with b'1111222244443333' **".format(offset))
         wbfr.seek(offset)
         wbfr.write(b'\x33\x33\x44\x44\x22\x22\x11\x11')
         wbfr.seek(offset)
         ReadMetadata(offset)
 
 
-# If you want to verify the written Metadata then run below segment of code
 def ReadMetadata(offset):
+    """
+    If you want to verify the written Metadata then run below segment of code
+    """
```

Comment:
"Verifies that meta-data contains the valid footer at the given offset."

Reply:
Acknowledge